src/fs/host.ts

```diff
@@ -87,18 +87,23 @@ export class IframeHost implements Host {
     private window: Window,
     private debounceDelay: number = 1_000
   ) {}
-  createStorage(logging: Logging): FSStorage {
+  createStorage(_logging: Logging): FSStorage {
     return new InMemoryFSStorage(undefined);
   }
-  async shouldReinitializeProject(storage: FSStorage): Promise<boolean> {
+  async shouldReinitializeProject(_storage: FSStorage): Promise<boolean> {
     // If there is persistence then it is the embedder's problem.
     return true;
   }
   createInitialProject(): Promise<PythonProject> {
     return new Promise((resolve) => {
-      this.window.addEventListener("load", () =>
-        notifyWorkspaceSync(this.parent)
-      );
+      // TODO: review this carefully!
```

Comment:
Does this still need reviewing carefully?

Reply:
This is OK (think it only happens in the unit tests anyway). Tried locally with classroom.
Will check classroom and Python integration again after merge.

Reply:
Comment removed in 9dc4ea0fe1556eee547d3ceba2e4f2a482a56186